Assets/Engine/Tile/TileRework/TileRestrictions.cs

```diff
@@ -1,8 +1,8 @@
-﻿using System.Collections;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using SS3D.Engine.Tiles;
 using UnityEngine;
 
-namespace SS3D.Engine.Tiles
+namespace SS3D.Engine.Tile.TileRework
```

Comment:
Honestly, we have had the new tilemap system for some time and the old system has been completely removed, I believe we can move the entire tilemap system 'up' one folder and do away with the "TileRework" folder.
(This can be saved for another issue though.)

Reply:
Yup, its on the list of things to do. Actually probably worth doing when working on #876